### ISSUE TITLE
Multiple Unnecessary UserWarnings on new project created with `rasa init`

### ISSUE THREAD

**Issue by TomV** [RasaHQ/rasa#8975]
**Rasa version**: 2.7.1

**Python version**: 3.7.10

**Operating system** (windows, osx, ...): Darwin-20.5.0-x86_64-i386-64bit

**Issue**: On a "virgin" rasa 2.7.1 project, after creating a new project with `rasa init`, running any of these core commands will result in UserWarnings described below:

*   `rasa test`
*   `rasa shell`
*   `rasa data validate`

I've already created a few pull requests to address these issues. They are in the review pipeline as I write this.

## UserWarnings

1.) model_confidence should be set to "linear_norm"

/rasa/utils/train_utils.py:605: UserWarning: model_confidence is set to `softmax`.

It is recommended to try using `model_confidence=linear_norm` to make it easier to tune fallback thresholds.

2.) evaluate_every_number_of_epochs > epochs (value =1!)

rasa/shared/utils/io.py:97:

UserWarning: the value of 'evaluate_every_number_of_epochs' is greater than the value of 'epochs'. No evaluation will occur.

This is, to say the least, a disconcerting error. Digging in further, I found that at the time the error the number set for "epochs" is 1 <span class="error">[!/images/icons/emoticons/warning.png!| /images/icons/emoticons/warning.png]</span>

3.) Only for `rasa data validate`: UserWarning 'good afternoon` both greeting and goodbye intent match.

.../rasa/shared/utils/io.py:97

UserWarning: The example 'good afternoon' was found labeled with multiple different intents in the training data. Each annotated message should only appear with one intent. You should fix that conflict The example is labeled with: goodbye, greet.

For this third error, my quick fix is to change the goodbye intent to replace "good afternoon" to "Have a good afternoon". However, that's not the ideal fix, since 'good afternoon' by itself may well be the intent "goodbye"

**Error (including full traceback)**: Not a Error, just a warnings, see above

**Command or request that led to error**: Create a new project using `rasa init` then cd into the new directory, and try any of these commands, and you will see one or both of the warnings above.

rasa train

rasa data validate

rasa shell

rasa test

**Content of configuration file (config.yml)** (if relevant):

default, which is sadly all commented settings commented out. (Unusual design decision, needs its own issue. My opinion is that settings should contain active values and be the "source of truth" for the project settings.)

<pre>(default config.yml)</pre>

**Content of domain file (domain.yml)** (if relevant):

<pre>default yml after clean "rasa init" (see goodbye and greeting intents)</pre>

**Comment by sara-tagger** [comment]
Thanks for raising this issue, @rctatman will get back to you about it soon✨

###### Please also check out the [docs](https://rasa.com/docs/) and the [forum](https://forum.rasa.com/) in case your issue was raised there too 🤗

**Comment by m-vdb** [comment]
Closing as stale
